third_party/rust-hypervisor-firmware-virtio/src/lib.rs

```diff
@@ -8,7 +8,12 @@
 
 #![no_std]
 
+use x86_64::{PhysAddr, VirtAddr};
+
 pub mod device;
 pub mod mem;
 pub mod pci;
 pub mod virtio;
+
+pub trait Translator: Fn(PhysAddr) -> Option<VirtAddr> {}
+impl<X: Fn(PhysAddr) -> Option<VirtAddr>> Translator for X {}
```

Comment:
See https://github.com/project-oak/oak/pull/3306#discussion_r992554251

Reply:
As mentioned there simple type aliases didn't work, so I had to do this two-step workaround.